<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MinecraftVersionsInterfaceClass</class>
 <widget class="QMainWindow" name="MinecraftVersionsInterfaceClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сохранение модов для Minecraft версий</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>resources/MineVersionLogo.ico</normaloff>resources/MineVersionLogo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="CurrentVersionGroup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Текущая версия</string>
    </property>
    <widget class="QLabel" name="CurrentMineVersion">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Версия игры:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Имя  версии:</string>
     </property>
    </widget>
    <widget class="QLabel" name="CurrentVersionName">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ClearMods">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>80</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Убрать моды</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UpdateModsB">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ClearCurentMod">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>151</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Отчистить
версию</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>771</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Список версий</string>
    </property>
    <widget class="QPushButton" name="SetVersion">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>40</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Загрузить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DeleteVersion">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>100</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QComboBox" name="VersionsList">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>40</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Имя  версии:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Версия игры:</string>
     </property>
    </widget>
    <widget class="QLabel" name="CurrentVersionName_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
    <widget class="QLabel" name="CurrentMineVersion_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>70</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>None</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>771</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Создать версию</string>
    </property>
    <widget class="QPushButton" name="MakeVersion">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>60</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Создать
версию</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Имя  версии:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Версия игры:</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="InputName">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="InputVersion">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>251</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="TextLogs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>771</width>
      <height>41</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Igorantivirus_v_beta">
    <property name="title">
     <string>Автор - Igorantivirus</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuv_0_5_beta">
    <property name="title">
     <string>v - 0.5 beta</string>
    </property>
   </widget>
   <addaction name="menu_Igorantivirus_v_beta"/>
   <addaction name="menuv_0_5_beta"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MinecraftVersionsInterface.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>ListIndexChanged(int)</slot>
  <slot>DeleteButtonPressed()</slot>
  <slot>ClearMods()</slot>
  <slot>MakeVersionPressed()</slot>
  <slot>UpdateMods()</slot>
 </slots>
</ui>
